### ISSUE TITLE
How can I run a training process on RTX 2070?

### ISSUE THREAD

**Issue by AbdulkadirGul** [Unity-Technologies/ml-agents#2170]
Hi,

I have RTX2070 GPU that only supports CUDA >= 10 versions.  When I start a training process i have error like "cudart64_90.dll" not found. How can I run a training process on RTX GPU?

Have a nice day :)

**Comment by xiaomaogy** [comment]
Hi @AbdulkadirGul , we don't support CUDA >= 10 due to tensorflow version constraint.

**Comment by caioc2** [comment]
You could build tensorflow from source with cuda 10, but that's will require you a good amount of work.

**Comment by AbdulkadirGul** [comment]
> You could build tensorflow from source with cuda 10, but that's will require you a good amount of work.
> 
> https://www.tensorflow.org/install/source_windows

@xiaomaogy @caioc2 ty for reply.

@caioc2 i tried to build tensorflow 1.7 with CUDA 10 but i stuck on :

> No default toolchain found for cpu 'x64 windows'.

I have AVX2 architecture processor, when i push /arch:AVX2 to bazel configure nothing changes.

Do you have a working version of tensorflow 1.7 with CUDA 10?

**Comment by xiaomaogy** [comment]
We don't= =

**Comment by caioc2** [comment]
I have used tensorflow 1.12 with ML-Agents 0.6, it worked at that time, maybe it may works with the current version (and it is probably easier to find/build with cuda 10). Sadly I don't have 1.7 built with cuda 10.

**Comment by AbdulkadirGul** [comment]
@caioc2 THANK YOU! ML-Agents 0.82 with tensorflow r1.12 works PERFECT! But still I can't build tensorflow from source :(

This is the steps I followed:

`1. Installed CUDA v10.0 and CUDNN v7.3.1`

`2. Downloaded r1.12 wheel from: https://github.com/fo40225/tensorflow-windows-wheel/tree/master/1.12.0/py36/GPU/cuda100cudnn73avx2`

`3. Created a new anaconda env than installed wheel with pip`

`4. Downgraded numpy to 1.14.5`

`5. Changed parameter in the ml-agents/ml-agents/setup.py:`

`>>>> "tensorflow>=1.7,<1.8" -> "tensorflow-gpu>=1.7"`

`6. Installed ML-Agents 0.82`

`7. Fingers crossed and works perfect!`
